modules/rte_mis_school/src/Access/SchoolPdfDownloadAccessCheck.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Drupal\rte_mis_school\Access;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Routing\Access\AccessInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Session\AccountProxyInterface;
+use Drupal\eck\EckEntityInterface;
+
+/**
+ * Determines edit access to the school details mini node.
```

Comment:
The comment doesn't match the class description.

Reply:
Modified.